packages/bsb-nest/bsb_nest/connection.py

```diff
@@ -35,10 +42,13 @@ class NestConnectionSettings:
     Class interfacing a NEST connection rule.
     """
 
-    rule = config.attr(type=str)
-    """Importable reference to the Nest connection rule used to connect the cells."""
-    constants = config.catch_all(type=types.any_())
-    """Dictionary of parameters to assign to the connection rule."""
+    rules = config.list(type=str)
+    """List of references to Nest connection rules used to connect the cells."""
```

Comment:
```suggestion
    """List of references to NEST connection rules used to connect the cells."""
```

I think this is how we stylize it.